### PR TITLE
Feat/beta warning

### PR DESCRIPTION
**Pull request by jleveugle**

### CODE REVIEW

File: packages/manager/apps/public-cloud/src/assets/theme/_typography.less [hunk 1]
```diff
@@ -6,3 +6,22 @@ h1 {
   color: @text-color;
   font-weight: 400;
 }
+
+h2 {
```
Comment (antleblanc) [inline packages/manager/apps/public-cloud/src/assets/theme/_typography.less]: Is applying theses changes to `.oui-heading_X` would be useful too?

File: packages/manager/apps/public-cloud/src/assets/theme/_typography.less [hunk 2]
```diff
@@ -6,3 +6,22 @@ h1 {
...
+}
+
+h3 {
+  font-size: 20px;
+  line-height: 25px;
+  font-weight: bold;
+  color: @text-color;
+}
+
+p {
```
Comment (antleblanc) [inline packages/manager/apps/public-cloud/src/assets/theme/_typography.less]: Is applying theses changes to `.oui-paragraph` would be useful too?

File: packages/manager/apps/public-cloud/src/assets/theme/_typography.less [hunk 3]
```diff
@@ -6,3 +6,22 @@ h1 {
   color: @text-color;
   font-weight: 400;
 }
+
+h2 {
+  font-size: 28px;
```
Comment (antleblanc) [inline packages/manager/apps/public-cloud/src/assets/theme/_typography.less]: `font-*` rules could be at least be grouped I think.

File: packages/manager/apps/public-cloud/src/components/beta-warning/beta-warning.controller.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import illustration from './beta-warning.png';
+
+const LOCALSTRORAGE_KEY = 'ovhManagerPublicCloud_hideBetaWarning';
```
Comment (antleblanc) [inline packages/manager/apps/public-cloud/src/components/beta-warning/beta-warning.controller.js]: I'm not convinced about using both camelCase and snakeCase.

For instance:
```suggestion
const LOCALSTRORAGE_KEY = 'OVH_MANAGER_PUBLIC_CLOUD_HIDE_BETA_WARNING';
```

File: packages/manager/apps/public-cloud/src/components/beta-warning/index.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import angular from 'angular';
+import 'ovh-ui-angular';
+import ngOvhSidebarMenu from '@ovh-ux/ng-ovh-sidebar-menu';
```
Comment (antleblanc) [inline packages/manager/apps/public-cloud/src/components/beta-warning/index.js]: Is `@ovh-ux/ng-ovh-sidebar-menu` really needed?

File: packages/manager/apps/public-cloud/src/index.html [hunk 6]
```diff
@@ -18,14 +18,14 @@
                         </div>
                         <div class="h-100 position-relative" data-ng-class="{ 'col-md-10 p-0': $ctrl.shouldDisplaySidebar, 'col-md-12': !$ctrl.shouldDisplaySidebar }">
                             <div class="position-absolute w-100 h-100 py-4 px-5" style="overflow: auto;">
-                                <ng-ovh-uirouter-breadcrumb></ng-ovh-uirouter-breadcrumb>
-                                <div class="mt-3" data-ui-view></div>
+                                    <ng-ovh-uirouter-breadcrumb></ng-ovh-uirouter-breadcrumb>
```
Comment (antleblanc) [inline packages/manager/apps/public-cloud/src/index.html]: Extra indentation.

File: packages/manager/apps/public-cloud/src/index.html [hunk 7]
```diff
@@ -18,14 +18,14 @@
...
+                                    <div class="mt-3" data-ui-view></div>
+                                </div>
```
Comment (antleblanc) [inline packages/manager/apps/public-cloud/src/index.html]: Extra closing `</div>` tag.

```suggestion
```

File: packages/manager/apps/public-cloud/src/index.html [hunk 8]
```diff
@@ -37,6 +37,7 @@ <h1 class="mt-3 mb-5" data-ng-if="('preload_manager_ovh' | translate) !== 'prelo
                     </div>
                 </div>
             </div>
+            <beta-warning></beta-warning>
         </div>
     </body>
-</html>
+    </html>
```
Comment (antleblanc) [inline packages/manager/apps/public-cloud/src/index.html]: Extra whitespace.

File: packages/manager/apps/public-cloud/src/components/beta-warning/index.js [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+import './beta-warning.less';
+
+const moduleName = 'ovh-manager-betaWarning';
+
+angular
+  .module(moduleName, [
+    'oui',
+    ngOvhSidebarMenu,
+  ])
+  .component('betaWarning', component)
```
Comment (antleblanc) [inline packages/manager/apps/public-cloud/src/components/beta-warning/index.js]: As this component is "Public Cloud" oriented, what do you think about prefixing it?

For instance:

```suggestion
  .component('publicCloudBetaWarning', component)
```

File: packages/manager/apps/public-cloud/src/components/beta-warning/beta-warning.controller.js (hunk 4 above)
Comment (marie-j) [reply packages/manager/apps/public-cloud/src/components/beta-warning/beta-warning.controller.js]:
```suggestion
const LOCALSTORAGE_KEY = 'ovhManagerPublicCloud_hideBetaWarning';
```
also :)

File: packages/manager/apps/public-cloud/src/components/beta-warning/beta-warning.html [hunk 10]
```diff
@@ -0,0 +1,39 @@
+<div class="p-4" data-ng-if="!$ctrl.isHidden">
+    <h1 class="d-flex logo">
+        <i class="oui-icon oui-icon-ovh" style="font-size: 5rem;"></i>
+        <span class="ml-3 my-auto">OVH</span>
+    </h1>
+    <div class="container-fluid">
+        <div class="row d-flex justify-content-center">
+            <div class="col-xs-10 col-md-6 col-lg-5">
+                <div class="text-center mb-4">
+                    <img class="mb-4" data-ng-src="{{ $ctrl.illustration }}" alt="" style="width: 8vw;" />
```
Comment (marie-j) [inline packages/manager/apps/public-cloud/src/components/beta-warning/beta-warning.html]: Empty `alt`? 
Also I think you can avoid inline style with `boostrap` utilities https://getbootstrap.com/docs/4.3/utilities/sizing/#relative-to-the-viewport

File: packages/manager/apps/public-cloud/src/components/beta-warning/index.js [hunk 11]
```diff
@@ -0,0 +1,19 @@
+import angular from 'angular';
+import 'ovh-ui-angular';
+import ngOvhSidebarMenu from '@ovh-ux/ng-ovh-sidebar-menu';
+
+import component from './beta-warning.component';
+
+import './beta-warning.less';
+
+const moduleName = 'ovh-manager-betaWarning';
```
Comment (marie-j) [inline packages/manager/apps/public-cloud/src/components/beta-warning/index.js]:
```suggestion
const moduleName = 'ovhManagerBetaWarning';
```
to keep consistency with other namings

File: packages/manager/apps/public-cloud/src/components/beta-warning/translations/Messages_fr_FR.json [hunk 12]
```diff
@@ -0,0 +1,11 @@
+{
+  "public_cloud_beta_warning_welcome": "Bienvenue !",
+  "public_cloud_beta_warning_subtitle": "Découvrez la version Preview de Public Cloud",
+  "public_cloud_beta_warning_test_new_feature": "Testez les nouvelles fonctionnalités",
+  "public_cloud_beta_warning_test_new_feature_desc": "Vous bénéficiez d’un accès anticipé à notre future interface de gestion de vos projets Public Cloud.",
+  "public_cloud_beta_warning_help_develop": "Aidez-nous à développer Public Cloud",
+  "public_cloud_beta_warning_help_develop_desc": "En utilisant la version Preview de Public Cloud et en partageant vos commentaires (à l’aide du bouton situé en bas à droite de votre manager), vous pourrez nous aider à améliorer les fonctionnalités.",
```
Comment (marie-j) [inline packages/manager/apps/public-cloud/src/components/beta-warning/translations/Messages_fr_FR.json]: `manager` -> `espace client` ?

File: packages/manager/apps/public-cloud/src/assets/theme/_typography.less (hunk 1 above)
Comment (jleveugle) [reply packages/manager/apps/public-cloud/src/assets/theme/_typography.less]: done

**Pull request merged**